scripts/prepare/bundle.ts

```diff
@@ -75,26 +102,25 @@ const run = async ({ cwd, flags }: { cwd: string; flags: string[] }) => {
   if (formats.includes('esm')) {
     tasks.push(
       build({
+        treeshake: true,
         silent: true,
         entry: allEntries,
         watch,
         outDir,
+        env: {
+          NODE_ENV: 'production',
+        },
         format: ['esm'],
         target: 'chrome100',
         clean: !watch,
         ...(dtsBuild === 'esm' ? dtsConfig : {}),
         platform: platform || 'browser',
-        esbuildPlugins: [
-          aliasPlugin({
```

Comment:
Is this something which is already built into rollup and we don't need this anymore?

Reply:
I'm not sure if this means:

> tsup offers an additional option to let you use Rollup for tree shaking instead:

esbuild won't be used at all. Or if rollup will be used for some part on top of esbuild...

Reply:
As I understand the code, rollup is used on top of esbuild as a kind of esbuild plugin. But let's be 100% sure :)

Reply:
Yes, got confirmation that the above information is correct